lib/SearchAndSort/SearchAndSort.js

```diff
@@ -5,8 +5,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Route from 'react-router-dom/Route';
 import { withRouter } from 'react-router';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Why the lint-disable?

Reply:
(I know you didn't add it in this PR, but it bothers me.)

Reply:
I might not need to be disabled, as on my machine eslint was throwing similar errors for imports that were not throwing errors on the ci linting. Or then again, maybe that the best time to use such comments, as it prevents false positives for the presence of errors when running liniting locally.

Reply:
OK. We have an ongoing issue with ESLint errors from inability to resolve imports from peer-dependencies, which we need to think about separately. Let's ignore this problem for now, and make a separate issue for it elsewhere.

Reply:
(I wish there was a way for me to mark this conversation thread as Resolved.)